backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/ExpiredPatientLinkException.java

```diff
@@ -0,0 +1,9 @@
+package gov.cdc.usds.simplereport.api.model.errors;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.GONE, reason = "This patient link is expired.")
```

Comment:
TIL that you can apply this annotation to the exceptions themselves—neat! (I've only used it before in an ExceptionHandler method.)